pkg/utils/util.go

```diff
@@ -109,6 +108,41 @@ var SnapshotterListSecretParams = secretParamsMap{
 	secretNamespaceKey: PrefixedSnapshotterListSecretNamespaceKey,
 }
 
+// ContainsString checks if a given slice of strings contains the provided string.
+// If a modifier func is provided, it is called with the slice item before the comparation.
+func ContainsString(slice []string, s string, modifier func(s string) string) bool {
```

Comment:
nit, can we remove the modifier? it appears its not used at all in this repo (i.e., all nils).

Reply:
done